beta-src/src/components/svgr-components/WDArmyIcon.tsx

```diff
@@ -2,93 +2,99 @@ import * as React from "react";
 import { useTheme } from "@mui/material/styles";
 import { gameIconProps } from "../../interfaces/Icons";
 import UIState from "../../enums/UIState";
+import WDUnitController from "../ui/WDUnitController";
 
 const WDArmyIcon: React.FC<gameIconProps> = function ({
   country,
   height = 50,
-  onClick = undefined,
   iconState = UIState.NONE,
+  meta,
   viewBox,
   width = 50,
 }): React.ReactElement {
   const theme = useTheme();
+  const [fluidIconState, setFluidIconState] = React.useState(iconState);
```

Comment:
import React, { useState } from 'react';

useState(iconState);

doesn't make a difference but so were not repeating React...

Reply:
Even though I do agree this would save us some words, I do this on purpose because the `React.` part is highlighted in green which allows me to see what functions are react right away.